test_pipeline.py

```diff
@@ -0,0 +1,144 @@
+                             "New data will be appended to these files.")
+  
+    parser.add_argument("json_output_path", metavar="json-output-path",
+                        help="Path to a JSON file to write TracedData for final analysis file to")
+    parser.add_argument("icr_output_dir", metavar="icr-output-dir",
+                        help="Directory to write CSV files to, each containing n messages and message ids for use " 
+                             "in inter-code reliability evaluation"),
+    parser.add_argument("coded_dir_path", metavar="coded-dir-path",
+                        help="Directory to write coded Coda files to")
+    parser.add_argument("csv_by_message_output_path", metavar="csv-by-message-output-path",
+                        help="Analysis dataset where messages are the unit for analysis (i.e. one message per row)")
+    parser.add_argument("csv_by_individual_output_path", metavar="csv-by-individual-output-path",
+                        help="Analysis dataset where respondents are the unit for analysis (i.e. one respondent "
+                             "per row, with all their messages joined into a single cell)")
+    parser.add_argument("production_csv_output_path", metavar="production-csv-output-path",
+                        help="Path to a CSV file to write raw message and demographic responses to, for use in "
+                             "radio show production"),
+
+    args = parser.parse_args()
+
+    drive_credentials_path = None
+    csv_by_message_drive_path = None
+    csv_by_individual_drive_path = None
+    production_csv_drive_path = None
+
+    drive_upload = args.drive_upload is not None
+    if drive_upload:
+        drive_credentials_path = args.drive_upload[0]
+        csv_by_message_drive_path = args.drive_upload[1]
+        csv_by_individual_drive_path = args.drive_upload[2]
+        production_csv_drive_path = args.drive_upload[3]
+
+    user = args.user
+
+    phone_number_uuid_table_path = args.phone_number_uuid_table_path
+    water_filter_input_path = args.water_filter_input_path
+    internet_working_input_path = args.internet_working_input_path
+    waste_disposal_input_path = args.waste_disposal_input_path
+    demog_input_path = args.demog_input_path
+    prev_coded_dir_path = args.prev_coded_dir_path
+
+    json_output_path = args.json_output_path
+    icr_output_dir = args.icr_output_dir
+    coded_dir_path = args.coded_dir_path
+    csv_by_message_output_path = args.csv_by_message_output_path
+    csv_by_individual_output_path = args.csv_by_individual_output_path
+    production_csv_output_path = args.production_csv_output_path
+
+    shows_paths = [water_filter_input_path, internet_working_input_path, waste_disposal_input_path]
+
+    # Load phone number <-> UID Table
+    print("Loading phone Number <-> UUID Table...")
+    with open(phone_number_uuid_table_path, "r") as f:
+        phone_number_uuid_table_path = PhoneNumberUuidTable.load(f)
+
+    #Load Messages
```

Comment:
Convention in Python is to put a space after the '#'. Python editors should be able to point this out to you automatically, you may need to flick a switch somewhere to enable "PEP-8" support